### ISSUE TITLE
Using Polarbear to predict scRNA-seq data for human brain cells

### ISSUE THREAD

**Issue by TedRees61** [Noble-Lab/Polarbear#2]
Hi Dr. Zhang,

I read your Polarbear paper and planned to run the model on some multiomics data for the Human Brain to see how well your model performs on human brain tissue. Do you have a pretrained model for the human brain or other suitable model? Are there pre-processing steps I would need for 10X Genomics multimodal data before I could apply your model? I assumed I could run such a model with just scATAC-Seq data to make scRNA-Seq predictions but the documentation seems to suggest otherwise, or is this just for model training?

Thanks for your help

Ted

**Comment by RanZhang08** [comment]
Hi Ted, thanks for your interest!

We unfortunately don't have pre-trained model on human brain. If you have access to some multimodal data (that have matched scRNA-seq and scATAC-seq from the same cell), you can train your own model with our code using that and any additional scRNA-seq and scATAC-seq profiles in human brain.

Our model takes in raw counts of scRNA-seq and binarized counts of scATAC-seq, and there are no further data normalization needed. Two potential simple preprocessing steps include: If your input contains multiple scATAC-seq datasets, you will need to match the peaks to a common set of coordinates before inputting them to our model. Also, you will need to make sure the peaks are sorted by chromosome in the scATAC-seq domain. More examples of the data can be found in https://noble.gs.washington.edu/~ranz0/Polarbear/data/.

The model can perform translation in either direction, so it's perfectly designed to input scATAC-seq profile and predict corresponding scRNA-seq. 

Please let us know if you have any further questions!
Ran

**Comment by TedRees61** [comment]
Hi Ran,

Thanks for the quick response. Unfortunately we only have one multimodal data set which we have been using for benchmarking purposes. Am i correct in thinking you have a pre-trained model for adult mouse brain only? 

Thanks again for your help

Ted

**Comment by RanZhang08** [comment]
Hi Ted,

Yes, you are correct. If you have a multimodal dataset where each cell is captured by both single-cell RNA-seq and ATAC-seq assays, then a new model can be trained using that data. Not sure if it helps?

Thanks,
Ran
